.github/workflows/tests.yml

```diff
@@ -102,6 +102,9 @@ jobs:
       - name: Force vendored Ruby on Catalina
         if: matrix.version == '10.15'
         run: echo 'HOMEBREW_FORCE_VENDOR_RUBY=1' >> $GITHUB_ENV
```

Comment:
This shouldn't be needed any more FYI.

Reply:
I think I remember the PR where you were meant to have fixed it, but CI was still acting up after that for some reason.